spec/support/features/doc_auth_helper.rb

```diff
@@ -90,7 +86,11 @@ def complete_doc_auth_steps_before_agreement_step(expect_accessible: false)
   end
 
   def complete_agreement_step
-    find('label', text: t('doc_auth.instructions.consent', app_name: APP_NAME)).click
+    find(
+      'label',
+      text: t('doc_auth.instructions.consent', app_name: APP_NAME),
+      wait: 20,
+    ).click
```

Comment:
From explorations during test hacking, I don't think the `wait: 20` is having any effect. Recommend removing this change and setting CAPYBARA_WAIT_TIME_SECONDS to a higher value, as described by @jmax-gsa [here](https://gsa-tts.slack.com/archives/C0NGESUN5/p1678478123069019).

Reply:
Capybara will not be able to find the label unless we put some kind of wait in here

Reply:
> Capybara will not be able to find the label unless we put some kind of wait in here

Why is that?

Reply:
I had assumed it's because it takes an extra bit of time for the elements on the page to load (due to React maybe?). Several of the helper methods that deal with clicking things have short `wait` calls -- [here is one example](https://github.com/18F/identity-idp/blob/main/spec/support/features/idv_helper.rb#L61)

Reply:
I'd agree that typically I'd only expect to have to wait if there were some dynamic content shown after page load, but the consent screen is pretty static, which is why it's surprising/alarming to me as well.

Reply:
There is a spinner button on the agreement page. I'm not sure if that's new or what -- maybe it has to do with checking for mobile on the frontend

Reply:
Ah, in that case, could we use the `click_spinner_button_and_wait` helper instead?

https://github.com/18F/identity-idp/blob/6d2a4527b37cf69ecc52ad42acc86ab3f8aeffa3/spec/support/features/interaction_helper.rb#L2-L5

Reply:
I found the issue is that the preceding step uses a spinner button but [the `complete_welcome_step`](https://github.com/18F/identity-idp/blob/a7817d94fa872d95c677a19f04dc54c4c154157c/spec/support/features/doc_auth_helper.rb#L83-L85) helper doesn't wait for the button to resolve. You only have to wait for the label because the welcome step may not have finished submitting by the time the assertion is made.

I've got a related pull request already in the works that I'll include a change for this.

Reply:
for posterity: https://github.com/18F/identity-idp/pull/8013